docs/requirements.txt

```diff
@@ -1,3 +1,12 @@
-sphinx_rtd_theme==0.4.3
-autodocsumm==0.2.3
-sphinxcontrib-napoleon==0.7
+sphinx==4.4.0
+nbsphinx==0.8.8
+autodocsumm==0.2.7
+sphinx-tabs==3.2.0
+furo==2022.1.2
+ipython==8.0.1
+ipykernel==6.8.0
+tqdm==4.62.3
+ipywidgets==7.6.5
+sphinx-multiversion==0.2.4
+cleanlab==1.0.0
```

Comment:
How is multiversion being handled?
I.e. stable (most recent pypi) vs latest (master branch)

Reply:
removed the line `cleanlab==1.0.0`